pyogrio/geopandas.py

```diff
@@ -306,25 +306,27 @@ def write_dataframe(
 
     path = str(path)
 
-    if not isinstance(df, gp.GeoDataFrame):
-        raise ValueError("'df' must be a GeoDataFrame")
+    if not isinstance(df, (gp.GeoDataFrame, pd.DataFrame)):
```

Comment:
```suggestion
    if not isinstance(df, pd.DataFrame):
```

Since GeoDataFrame is a subclass, isinstance works with DataFrame as well

Reply:
ok